### ISSUE TITLE
Use github releases and pypi

### ISSUE THREAD

**Issue by zommuter** [mingwpy/mingwpy#26]
See [here](https://help.github.com/articles/creating-releases/). That would be a logical alternative to https://anaconda.org/carlkl/mingwpy

**Comment by techtonik** [comment]
Makes sense.

**Comment by carlkl** [comment]
An even better alternative is to use pypi.io when it is time to release.

**Comment by zommuter** [comment]
@carlkl Good point - am I correct assuming http://pypi.io is linked to https://pypi.python.org/pypi, i.e. one could then simply `pip install mingwpy`?
